src/features/people/PersonPage/index.js

```diff
@@ -1,3 +1,179 @@
+import { SubHeaderDate, SubHeaderPerson, CastCrewHeader } from "./styled";
+
 export const PersonPage = () => {
-  return <div>strona Person</div>;
+  const dispatch = useDispatch();
+
+  const { id } = useParams();
+
+  useEffect(() => {
+    dispatch(fetchPersonPage(id));
+    dispatch(fetchPersonDetailsPage(id));
+  }, [dispatch, id]);
+
+  const cast = useSelector(selectCast);
+  const crew = useSelector(selectCrew);
+  const person = useSelector(selectPerson);
+
+  return (
+    <>
+      {person.map(
+        ({ profile_path, name, birthday, place_of_birth, biography }) => (
+          <MainContainer>
+            <Image src={`${imageBaseUrl}/w342${profile_path}`} />
+            <TileContainer>
+              <BigTileTitle>{name}</BigTileTitle>
+              <SubHeaderContainer>
+                <SubHeaderPerson>
+                  <SubHeaderDate>date of birth:</SubHeaderDate>
+                  <SubHeaderInformation>{birthday}</SubHeaderInformation>
+                  <SubHeaderDate>
+                    {place_of_birth ? "place of birth:" : null}
+                  </SubHeaderDate>
+                  <SubHeaderInformation>{place_of_birth}</SubHeaderInformation>
+                </SubHeaderPerson>
+              </SubHeaderContainer>
+            </TileContainer>
+            <Article>{biography}</Article>
+          </MainContainer>
+        )
+      )}
+      <CastCrewHeader>
+        movies - cast {`(`}
+        {cast.length}
+        {`)`}
+      </CastCrewHeader>
+      <Wrapper>
+        {cast.map(
+          ({
+            poster_path,
+            id,
+            title,
+            character,
+            release_date,
+            vote_average,
+            vote_count,
+          }) => (
+            <>
+              <Container key={id}>
+                <Poster
+                  src={
+                    poster_path
+                      ? `${imageBaseUrl}/w342${poster_path}`
+                      : noPoster
+                  }
+                  alt=""
+                ></Poster>
+                <InfoContent>
+                  <Details>
+                    <Title>{title}</Title>
+                    <Year>
+                      {character} {character ? release_date : null}
+                    </Year>
+                    <GenreContainer>
+                      <GenreItem>Action</GenreItem>
+                      <GenreItem>Adventure</GenreItem>
+                    </GenreContainer>
+                  </Details>
+                  <RatingsContainer>
+                    <StyledStar />
+                    <Rate>{vote_average}</Rate>
+                    <MaxRating>/ 10</MaxRating>
+                    <Votes>{vote_count} votes</Votes>
+                  </RatingsContainer>
+                </InfoContent>
+              </Container>
+            </>
+          )
+        )}
+      </Wrapper>
+      <CastCrewHeader>
+        {crew.length > 0 ? `movies - crew  ${"("}${crew.length}${")"}` : null}
```

Comment:
ogólnie to super, że jest uwzględniony brak crew, ale wydaje mi się, że najlepiej będzie zrobić warunek logiczny do całości, a nie tylko do header, czyli np ```crew.length > 0 && ( i tutaj wrzucić wtedy wszystko w React Fragment), bo w tym wypadku jesli nie bedzie crew to i tak sie wyrenderuje header, tylko bedzie pusty :) i zastanówmy się czy każda osoba będzie miała cast - bo może warto też renderować pole z cast warunkowo